heartbeat/hbtestllext/isdefs.go

```diff
@@ -74,7 +74,7 @@ var IsMonitorStateInLocation = func(locName string) isdef.IsDef {
 		}
 
 		if !stateIdMatch.MatchString(s.ID) {
-			return llresult.SimpleResult(path, false, fmt.Sprintf("ID %s does not match regexp pattern /%s/", s.ID, locPattern))
+			return llresult.SimpleResult(path, false, "ID %s does not match regexp pattern /%s/", s.ID, locPattern)
```

Comment:
Are you sure about this change? Does `llresult.SimpleResult` support formatting?

Reply:
Yes, it passes the third argument onwards directly to fmt.Sprintf:

```go
// SimpleResult provides a convenient and simple method for creating a *Results object for a single validation.
// It's a very common way for validators to return a *Results object, and is generally simpler than
// using SingleResult.
func SimpleResult(path llpath.Path, valid bool, msg string, args ...interface{}) *Results {
	vr := ValueResult{valid, fmt.Sprintf(msg, args...)}
	return SingleResult(path, vr)
}
```